Akkadu/rsi-api-widgets
# Feature/agora consumer

---
Please describe the purpose of the pull request.

* The purpose is to implement our current webapp language channel on the interpretation-player widget.
* And also that task : `Interpretation-player Widget, add logic to get the roomName from the query url`
  * [notion page](https://www.notion.so/akkadu/RSI-API-Interpretation-player-Widget-add-logic-to-get-the-roomName-from-the-query-url-fe43e7c144d44351aa8e5b9d480ed2c3)

## Motivation and Context


## How To Test
* Unfortunately we cannot use the Vue Preview since our api whitelist doesn't contain our deploy preview from `deploy-preview-7--rsi-akkadu-vue-demo`,
* Get the repo and go to this branch, then in the root do : `yarn install` and `yarn build`
  * go to `packages/rsi-api-vue` and run `yarn start`, you should get the vue preview. 
  * In the URL parameter you should see a parameter call `rsi-roomname`, 
    * Create an event (on the dev-api) and remplace that `rsi-roomname` by this new roomname.
    * Add an interpreter to this event and start interpreting.
    * On the vue-preview : 
      * When switching to the `source` language the video should be `unmuted` and our stream muted
      * When switching to the `target` language, the video should be `muted` and our stream `unmuted`
     * On the interpreter page:
       * If you change change the output language it should be updated as well on the vue-preview
     * If the stream gets stuck, you should see a `refresh button` 

* I also updated the [DOC](https://deploy-preview-7--rsi-akkadu-documentation.netlify.app/)

## Screenshots (if appropriate)

## Checklist



加油！
